Units/clojure.d/input.clj

```diff
@@ -0,0 +1,10 @@
+(ns app.controller)
+
+(defn empty-fn [])
+
+(defn function-with-body []
+    (println "body"))
+
+'(defn quoted-function [])
+(quote quoted-function2 [])
```

Comment:
quote (and it's syntax sugar ') prevents the code from evaluating: http://clojure.org/special_forms#quote so this shouldn't be inside the tags